pkg/scheduler/apis/config/v1alpha1/conversion.go

```diff
@@ -53,7 +53,7 @@ func Convert_v1alpha1_KubeSchedulerConfiguration_To_config_KubeSchedulerConfigur
 		}
 	}
 	if in.HardPodAffinitySymmetricWeight != nil {
-		args := interpodaffinity.Args{HardPodAffinityWeight: in.HardPodAffinitySymmetricWeight}
+		args := config.InterPodAffinityArgs{HardPodAffinityWeight: in.HardPodAffinitySymmetricWeight}
```

Comment:
This is a hard call, given that this is v1alpha1.
I think you can just hardcode the raw configuration with `fmt.Sprintf`.

Reply:
It would render the same result effectively. Do you think it's bad, because `v1alpha1` depends on `v1alpha2`?

Reply:
yes, let's not add that dependency